src/locale/locales/pt-BR/messages.po

```diff
@@ -3862,7 +3862,7 @@ msgstr "Listas de Moderação"
 
 #: src/components/moderation/LabelPreference.tsx:247
 msgid "moderation settings"
-msgstr ""
+msgstr "Configurações de Moderação"
```

Comment:
I think this is supposed to be all lowercase, and line 3869 capitalized

Reply:
yes

Reply:
yes